packages/components/organisms/f-checkout/src/services/addressService.js

```diff
@@ -126,9 +116,8 @@ function getAddressCoordsFromLocalStorage () {
     return null;
 }
 
-
 /**
- * Checkes whether the address values in local storage match what's is in the form.
+ * Checks whether the address values in local storage match what's is in the form.
```

Comment:
Going for your "Leave the code tidy" Boy Scout badge?

Reply:
lol, always `sir`